server/views/formBuilder.njk

```diff
@@ -255,11 +255,6 @@
             </ul>
         </div>
     </div>
-    <div class="govuk-body">
-        <div>
-            {% set removeFileUploadLink = "/licence/create/id/"+ licenceId +"/condition/id/" + condition.id + "/remove-upload?_csrf=" + csrfToken %}
-        </div>
```

Comment:
unused variable, I guess it was intended to be used but rather than fix now, would be better to just add support for what it was doing after this area stabilises (if still necessary)